js/app/Visualization/UI/style.less

```diff
@@ -626,7 +626,7 @@ hyperwall is: 3 1280px screens wide and 3 720pxs screens high.
   max-width: 551px;
   height: 100%;
   padding-bottom: 135pt;
-  z-index: 10000;
+  z-index: 1;
```

Comment:
@redhog the `z-index: 10000` seems to be something we may have added on the past due to some other bug. Have you tested this out locally? Please try it out, I don't want to merge this and have to solve another issue.